src/lib/github.js

```diff
@@ -1,9 +1,9 @@
-const Octokit = require("@octokit/rest");
```

Comment:
Woah! What happened to this file?

Reply:
Oh dear, I was using single quotes and then was like oh double quotes, why don't I change you to single quotes for consistency.

Reply:
Ahh, we should figure out how to keep quotes consistent. See: https://github.com/Financial-Times/github/commit/b5325968bd7577aa47932e9e206e3f028f634cc7

Reply:
haha I must have missed the memo about double quotes over single quotes, good to know!

Reply:
Well, that's a good point. This project (and a few of our other new ones) have been formatted with Prettier, which by default uses whichever quotes it things are best, depending on the contents of the string. I'm personally fine with this, but it's definitely different to [how things are in Next-land](https://github.com/Financial-Times/n-gage/blob/master/dotfiles/.eslintrc.js#L28) (quotes must always be single).